backend/src/main/java/ddangkong/facade/balance/content/BalanceCategoriesResponse.java

```diff
@@ -0,0 +1,15 @@
+package ddangkong.facade.balance.content;
+
+import ddangkong.domain.balance.content.Category;
+import java.util.List;
+
+public record BalanceCategoriesResponse(
+        List<String> categories
+) {
+    public static BalanceCategoriesResponse from(List<Category> categories) {
```

Comment:
의견)
```suggestion
) {

    public static BalanceCategoriesResponse create(List<Category> categories) {
```
컨벤션 확인 부탁드립니다!

Reply:
하핫;; 바로 반영하겠습니다!